<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BranchBox</class>
 <widget class="QFrame" name="BranchBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="conditionsWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="returnLabel">
       <property name="text">
        <string>RETURN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ExpressionLineEdit" name="returnValue">
       <property name="placeholderText">
        <string>Expression to calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="errorCheck">
       <property name="text">
        <string>Error</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="addConditionButton">
         <property name="text">
          <string>Add Condition</string>
         </property>
         <property name="icon">
          <iconset theme="list-add"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeBranchButton">
         <property name="text">
          <string>Remove Branch</string>
         </property>
         <property name="icon">
          <iconset theme="list-remove"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpressionLineEdit</class>
   <extends>QLineEdit</extends>
   <header>expressionlineedit.h</header>
   <slots>
    <signal>expressionUpdated(QString)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
